packages/office-ui-fabric-react/src/components/SwatchColorPicker/SwatchColorPicker.tsx

```diff
@@ -89,8 +76,6 @@ const SwatchColorPickerOption: React.StatelessComponent<ISwatchColorPickerOption
     let {
```

Comment:
Something I learned in fabric-internal is that if you are doing logic with your callbacks and you need to create separate functions like we are doing here, we shouldn't use StatelessComponent.